match2/commons/match_group_display_matchlist.lua

```diff
@@ -1,217 +1,291 @@
-local Matchlist = {}
+local Class = require('Module:Class')
+local DisplayHelper = require('Module:MatchGroup/Display/Helper')
+local DisplayUtil = require('Module:DisplayUtil')
+local Json = require('Module:Json')
+local LuaUtils = require('Module:LuaUtils')
+local MatchGroupUtil = require('Module:MatchGroup/Util')
+local Table = require('Module:Table')
+local TypeUtil = require('Module:TypeUtil')
 
-local DisplayHelper = require("Module:MatchGroup/Display/Helper")
-local OpponentDisplay = require("Module:OpponentDisplay")
-local MatchSummary = require("Module:MatchSummary")
-local Json = require("Module:Json")
-local HasDetails = require("Module:Brkts/WikiSpecific").matchHasDetails
-
-local getArgs = require("Module:Arguments").getArgs
-local utils = require("Module:LuaUtils")
 local html = mw.html
 
-local _frame
+local Matchlist = {propTypes = {}, types = {}}
 
-function Matchlist.get(frame)
-	local args = getArgs(frame)
+-- Called by MatchGroup/Display
+function Matchlist.luaGet(_, args)
```

Comment:
Why is the first param ignored? Module:Class?

Reply:
The first param is `frame` which I'm not using because I'm adopting the convention of using `mw.getCurrentFrame()` where it's needed